src/qibo/models/mitiqinterface.py

```diff
@@ -0,0 +1,51 @@
+from cirq import Circuit as cirqCircuit
```

Comment:
Please move this example to the example's folder, otherwise we have to include cirq as dependency.